nncf/common/pruning/utils.py

```diff
@@ -270,12 +305,14 @@ def count_flops_for_nodes(graph: NNCFGraph,
         num_out_channels = output_channels.get(name, node.module_attributes.out_channels)
         flops[name] = 2 * np.prod(node.module_attributes.kernel_size) * \
                       num_in_channels * num_out_channels * np.prod(output_shapes[name])
+        weights[name] = np.prod(node.module_attributes.kernel_size) * num_in_channels * num_out_channels
 
     for node in graph.get_nodes_by_metatypes(linear_op_metatypes):
         name = node.node_name
         flops[name] = 2 * np.prod(input_shapes[name]) * np.prod(output_shapes[name])
+        weights[name] = np.prod(input_shapes[name]) * np.prod(output_shapes[name])
 
-    return flops
+    return flops, weights
```

Comment:
Seems like the tuple `(flops, weights)` is not a `Tuple[Dict[str, int], Dict[str, int]]` because `np.prod()` method returns numpy type. For example,
```python
xs = [1, 2, 3]
ans = np.prod(xs)
isinstance(ans, int)  # False
```
I'm not sure that it's a big problem. @alexsu52 What do you think about this?

Reply:
I agree that type-hint is not correct. @evgeniya-egupova, could you fix it?

Reply:
Done